addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/model/OshiSysteminfo.java

```diff
@@ -522,13 +583,19 @@ public DecimalType getCpuThreads() {
 
     @Override
     public StringType getNetworkMac(int networkIndex) throws DeviceNotFoundException {
+        // In the current OSHI version a new query is required for the network data values to be updated
+        // In OSHI 4.0.0. it is planed to change this mechanism - see https://github.com/oshi/oshi/issues/310
+        networks = hal.getNetworkIFs();
```

Comment:
Same as above.

Reply:
Might this have the consequences that if not all fields bound a refresh might never occur, should we take look at some expiring cache instead here?